src/components/display/fewlines/IdentitiesSection/IdentitiesSection.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import styled from "styled-components";
+
+import { Button, ButtonVariant } from "../Button/Button";
+import { NeutralLink } from "../NeutralLink";
+import { Separator } from "../Separator/Separator";
+import { ShadowBox } from "../ShadowBox/ShadowBox";
+import { ShowMoreButton } from "../ShowMoreButton/ShowMoreButton";
+import { Timeline } from "../Timeline/Timeline";
+import { TimelineEnd } from "../TimelineEnd/TimelineEnd";
+import { Identity } from "@lib/@types";
+
+type IdentitieSectionProps = {
```

Comment:
Let me fix this spelling mistake: `IdentitiesSectionProps` instead of `IdentitieSectionProps`

Reply:
fixed 4335287